enterprise/com/src/main/java/org/neo4j/com/storecopy/TransactionCommittingResponseUnpacker.java

```diff
@@ -188,6 +196,13 @@ private void applyQueuedTransactions() throws IOException
             // Check rotation explicitly, since the version of append that we're calling isn't doing that.
             logRotation.rotateLogIfNeeded( LogAppendEvent.NULL );
 
+            // Check kernel health after log rotation
+            if ( !kernelHealth.isHealthy() )
+            {
+                log.error( msg );
```

Comment:
The error is logged when we call `kernelHealth.panic(...)`, here we don't know what went wrong... we simply know that the kernel is not healthy.